.github/workflows/broken-link-check.yml

```diff
@@ -21,7 +21,6 @@ jobs:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
 
       - name: Close Previous Link Checker Report GitHub Issue with label 'link-checker-report'
-        if: steps.lychee.outputs.exit_code != 0
```

Comment:
On success we were not removing the [Link Checker Issue.](https://github.com/hashicorp/web-unified-docs/actions/runs/13948638406/job/39041978626#step:3:39933)